tests/Test.ADAL.NET.Integration/DeviceProfileTests.cs

```diff
@@ -88,5 +88,51 @@ public void Initialize()
             Assert.AreEqual("some-access-token", result.AccessToken);
         }
 
+        [TestMethod]
+        public async Task FullCoveragePositiveTest()
+        {
+
+            MockHttpMessageHandler mockMessageHandler = new MockHttpMessageHandler()
+            {
+                Method = HttpMethod.Get,
+                Url = "https://login.microsoftonline.com/home/oauth2/devicecode",
```

Comment:
use TestConstants to use "home" tenant. there should be a constant called TestConstants.DefaultAuthorityHomeTenant